src/librbd/internal.cc

```diff
@@ -124,25 +124,25 @@ bool compare_by_name(const child_info_t& c1, const child_info_t& c2)
 } // anonymous namespace
 
   int detect_format(IoCtx &io_ctx, const string &name,
-		    bool *old_format, uint64_t *size)
+                    bool *old_format, uint64_t *size)
```

Comment:
Nit: try to avoid changing tabs to spaces -- unless the lines are directly affected by the PR. While I prefer spaces, technically tabs are allowed in the project.